src/pip/_internal/wheel.py

```diff
@@ -1036,14 +1037,22 @@ def build(
         # type: (...) -> List[InstallRequirement]
         """Build wheels.
 
-        :param unpack: If True, replace the sdist we built from with the
-            newly built wheel, in preparation for installation.
+        :param autobuilding: If True, replace the sdist we built from with the
```

Comment:
Rather than making this change, how about using this PR also as an opportunity to rename `autobuilding` to something more self-descriptive like `should_unpack` (as I suggested [here](https://github.com/pypa/pip/pull/6853#issuecomment-520610634)). That way the argument better describes what it does and will better match the semantics of what we want going forward. (And judging by the docstring, it looks like the original author was thinking about using something like that as the name, too, anyways.) The word "auto-building" still seems confusing to me. :) Also, I'm suggesting `should_unpack` rather than e.g. `unpack` because it will be a unique string in the code base, making it easy to search.

Reply:
Also, maybe the argument description can be clarified to say "If True, after the building the wheel, unpack it and replace the sdist with the unpacked version in preparation for installation." That way it's clear that not just the replacement but also the unpacking only happens if the argument is true

Reply:
Done